proposals/NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md

```diff
@@ -0,0 +1,103 @@
+# Addition of the way to notify HMI of the required support for apps
```

Comment:
Thank you for your advice.
We think "Add Notifications for Required App Support" is made a sense.
We will modify the title name.
Also, we will change the file name to "NNNN-Add-Notifications-for-Required-App-Support.md".
If you have any problem, please teach us.
Thank you.

Reply:
@Akihiro-Miyazaki by changing the file name, I am unable to review the specific changes you've made to the original file. Can you please change the file name back to its original so that I can view the diff to verify that all of my feedback has been addressed?

Reply:
Thank you! Please also change the PR name to match the proposal name: Add Notifications for Required App Support

Reply:
@theresalech -san,
Thank you for your advice.
We did not know the proposal name can be changed.
We changed the name.
Thank you.